main.go

```diff
@@ -73,7 +73,16 @@ func parseFile(fileName string, romaji bool) {
 		i++
 	}
 
-	newFileName := strings.Split(fileName, ".")[0] + "(parsed).md"
+	path := "parsed"
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		os.Mkdir(path, os.ModePerm)
+	}
+
+	if err := os.Chdir(path); err != nil {
+		panic(err)
+	}
+
+	newFileName := strings.Split(filepath.Base(fileName), ".")[0] + ".romanji.md"
```

Comment:
Since we are parsing from romaji to hiragana, the filename should probably be hiragana instead of romaji

Reply:
Hi @syentix, I pushed the change in this [commit](https://github.com/syentix/japanese-md-parser/pull/4/commits/8535ecd23961bb1175262194bba04cdd9d507c7f). Also, I updated the`README` with the new changes.